spec/expression.dd

```diff
@@ -86,6 +137,85 @@ functions with `extern (D)` $(DDSUBLINK spec/attribute, linkage, linkage).)
     $(BEST_PRACTICE Even though the order of evaluation is well-defined, writing code that
     depends on it is rarely recommended.)
 
+$(H2 $(LNAME2 temporary-lifetime, Lifetime of Temporaries))
+
+$(P Expressions and statements may create and/or consume rvalues. Such values are called
+$(I temporaries) do not have a name or a visible scope. Their lifetime is managed automatically as
```

Comment:
I'm not sure about the grammar in this sentence. Maybe we should insert an "and" after "temporaries"?

Reply:
Thanks, fixed.